### ISSUE TITLE
Visualizer_rllib displaying nan values for mean speed

### ISSUE THREAD

**Issue by mightyking7** [flow-project/flow#794]
## Bug Description

I used flow/visualizer/visualizer_rllib.py to replay a trained policy from a set checkpoint. The simulation progressed as expected and the reward and flow rate statistics were reported as expected. However, both the average and std deviation for the speed were NaN.

## Bug Reproduce

File in which the bug occurred: `flow/visualizer/visualizer_rllib.py`

The exact command that you did run that caused the bug:
`flow/visualizer/visualizer_rllib.py ~/ray_results/<Path to checkpoints dir>`

## Screenshots
![nan_bug](https://user-images.githubusercontent.com/16459029/70753092-95918d80-1cf9-11ea-9ef2-485427150197.png)

**Comment by eugenevinitsky** [comment]
@mightyking7 didn't you just comment that you fixed this on your end? Am I missing something? (Please tell me if I am so we can triage this properly).

**Comment by mightyking7** [comment]
Yes, I fixed it on my end. I created an issue to apply the fix to the main flow project as well.
